shared/constants/types/teams.tsx

```diff
@@ -114,38 +114,42 @@ export type TeamRoleMap = {
   loadedVersion: number
   roles: Map<TeamID, TeamRoleAndDetails>
 }
-
-export type State = Readonly<{
-  addUserToTeamsState: AddUserToTeamsState
-  addUserToTeamsResults: string
-  canPerform: Map<TeamID, TeamOperations>
+type Errors = {
```

Comment:
i think pulling this out doesn't really help discoverability. it'll make TS popovers more obscure etc. if you want to 'collect' them maybe preview with error* instead